control_flow.ipynb

```diff
@@ -10,24 +10,35 @@
   },
   {
    "cell_type": "markdown",
-   "id": "7d456e6b-159d-4959-b258-60963c381ff6",
+   "id": "5a1222fe",
    "metadata": {},
    "source": [
-    "## Table of contents\n",
-    "\n",
-    "- [References](#References-{visibility=\"hidden\"})\n",
-    "- [Conditionals](#Conditionals)\n",
-    "- [The `while` loop](#The-while-loop)\n",
-    "- [The `for` loop](#The-for-loop)\n",
-    "- [Nested loops](#Nested-loops)\n",
-    "- [Altering loops](#Altering-loops)\n",
-    "- [Exceptions](#Exceptions)\n",
-    "- [The `try-except` block](#The-try-except-block)\n",
+    "# Table of contents\n",
+    "- [Control flow](#Control-flow)\n",
+    "  - [Table of contents](#Table-of-contents)\n",
+    "  - [References](#References)\n",
+    "  - [Conditionals](#Conditionals)\n",
+    "  - [The `while` loop](#The-while-loop)\n",
+    "  - [The `for` loop](#The-for-loop)\n",
+    "    - [The `enumerate` built-in](#The-enumerate-built-in)\n",
+    "    - [The `range` built-in](#The-range-built-in)\n",
+    "  - [Nested loops](#Nested-loops)\n",
+    "  - [Altering loops](#Altering-loops)\n",
+    "    - [`if` statement inside `for`/`while`](#if-statement-inside-for/while)\n",
+    "    - [The `break` keyword](#The-break-keyword)\n",
+    "    - [The `continue` keyword](#The-continue-keyword)\n",
+    "    - [`else` after a `for`/`while`](#else-after-a-for/while)\n",
+    "  - [Exceptions](#Exceptions)\n",
+    "  - [The `try-except` block](#The-try-except-block)\n",
```

Comment:
```suggestion
    "- [References](#References)\n",
    "- [Conditionals](#Conditionals)\n",
    "- [The `while` loop](#The-while-loop)\n",
    "- [The `for` loop](#The-for-loop)\n",
    "    - [The `enumerate` built-in](#The-enumerate-built-in)\n",
    "    - [The `range` built-in](#The-range-built-in)\n",
    "- [Nested loops](#Nested-loops)\n",
    "- [Altering loops](#Altering-loops)\n",
    "    - [`if` statement inside `for`/`while`](#if-statement-inside-for/while)\n",
    "    - [The `break` keyword](#The-break-keyword)\n",
    "    - [The `continue` keyword](#The-continue-keyword)\n",
    "    - [`else` after a `for`/`while`](#else-after-a-for/while)\n",
    "- [Exceptions](#Exceptions)\n",
    "    - [The `try-except` block](#The-try-except-block)\n",
```

Adjust the TOC. No need to include a link to the TOC itself, I think. **Double-check** that it renders correctly.